src/twisted/python/test/test_deprecate.py

```diff
@@ -375,6 +375,29 @@ def callTestFunction():
     b = testFunction()
     if b == 3:
         deprecate.warnAboutFunction(testFunction, "A Warning String")
+"""
+        )
+        self.package.child("pep626.py").setContent(
+            b"""
+"A module string"
+
+from twisted.python import deprecate
+
+def noop():
+    pass
+
+def testFunction(a=1, b=1):
+    "A doc string"
+    if a:
+        if b:
+            noop()
+        else:
+            pass
```

Comment:
Do we need a real implementation for this test?
 
```suggestion
def testFunction():
    "The target for which the deprection is raised."

```

Reply:
Yes, otherwise it won't test the chamge

Reply:
Can we have a docstring that explain why this function needs to be structured this way and why a simpler one will not help with the test?

Just by looking at the 2 modules I can't spot the difference and I would say that they are duplicate tests :)